### PR TITLE
New package: dragon-1.0.0

### PR DESCRIPTION
**Pull request by TeddyDD**
Dragon is a little tool that allows using drag&drop from terminal :)
https://github.com/mwh/dragon

### CODE REVIEW

File: srcpkgs/dragon/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'dragon'
+pkgname=dragon
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="gtk+3-devel"
+short_desc="Simple drag-and-drop source/sink for X"
+maintainer="Daniel Lewan <vision360.daniel@gmail.com>"
+license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/dragon/template]: please use SPDX Identifier https://spdx.org/licenses/

Comment (maxice8) [reply srcpkgs/dragon/template]: GPL-3.0-or-later

**Comment by maxice8** [comment]
Please squash them into only 1 commit.

**Comment by TeddyDD** [comment]
moved to new repo

**Pull request closed without merging**